PD-343: Modify IBM cloud doc to include KVDB option (#678)
portworx/pxdocs: content/portworx-install-with-kubernetes/cloud/ibm/_index.md

In order for your Portworx cluster to be highly available, you must ensure that the Portworx key-value store is set up to be highly available. By using an instance of [Databases for etcd](https://cloud.ibm.com/docs/containers?topic=containers-portworx#portworx_database), you can set up a highly available key-value store for your Portworx cluster. Each service instance contains three etcd data members that are added to a cluster. The etcd data members are spread across zones in an IBM Cloud location and data is replicated across all etcd data members.

> Note that you can use the `Portworx metadata key-value store` option to specify if you want to install Portworx with:

Reviewer: Statements leading to bullet lists must be complete sentences. Although, I think we can omit the bullet list here, since the section above talks about the other option already. 

```suggestion
Alternatively, if you want to get up and running quickly, and don't need your key-value store to be highly available, you can select an internal KVBD from the `Portworx metadata key-value store` option on the deployment page.
```

> * The IBM hosted etcd or

Reviewer: ```suggestion
```

> * The internal KVDB.

Reviewer: ```suggestion
```

> If you choose to install Portworx with the internal KVDB, then you can skip the steps that ask for the endpoints and the secret. 

Reviewer: place this into a paragraph with the first sentence.

Author: OK
